### DISCUSSION TITLE
Does Cypress works for Webview?

### DISCUSSION THREAD

**Discussion by baeharam** [cypress-io/cypress#14327] [Questions and Help]
I have used Cypress for normal website, but I started new project for Webview(Android, iOS). Does Cypress is valid for Webview too? Is there guide for Webview?

**Comment by estefafdez** [comment]
I think cypress is not able to run tests on native mobile apps sadly.

**Reply by baeharam** [reply]
Cypress cannot run test in native mobile app. My question was that is cypress proper for Webview testing. I researched and concluded that just test mobile-screen webpage using Cypress. Anyway thanks for reply
